main.js

```diff
@@ -65,6 +65,13 @@ function createWindow () {
     }
   }, windowConfig);
 
+  if (windowOptions.fullscreen === false) {
+    delete windowOptions.fullscreen;
+  }
+
+  logger.info('Initializing BrowserWindow');
+  logger.info(windowOptions);
```

Comment:
If the first parameter passed to `bunyan` is an object, that object is merged with the final JSON log data: https://github.com/trentm/node-bunyan#log-method-api

So if we do things this way, we may want to construct our final uploaded log info a little differently.

Or we might just always pass strings and do something like a `util.inspect()` call on objects to turn them into strings...